app/js/lib/foxbox/foxbox.js

```diff
@@ -355,7 +355,7 @@ export default class Foxbox extends Service {
     return this[p.net].fetchJSON(`${this[p.net].origin}/services/${id}/state`)
       .then(res => {
         if (!res) {
-          throw new Error(`The action couldn't be performed.`);
+          throw new Error('The action couldn\'t be performed.');
```

Comment:
It's a shame because single and double quotes are frequently found in static strings.
I've always wanted to use a coding style that enforces the use of back tick as a string delimiter regardless of the presence of a variable inside the string.
But I understand your point so let's stick with single quote and back tick only if there's a parameter.